yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+  - name: exposure_id
+    "@id": "#RebExposureEFD.exposure_id"
+    datatype: long
+    nullable: false
+    description: Reference to the corresponding entry in the Exposure table.
+  - name: reb
+    "@id": "#RebExposureEFD.reb"
+    datatype: long
+    nullable: false
+    description: Rear Electronics Board ID.
+  - name: Camera_anaI
+    "@id": "#RebExposureEFD.Camera_anaI"
+    datatype: double
+    description: Current in mA
+  - name: Camera_anaV
+    "@id": "#RebExposureEFD.Camera_anaV"
+    datatype: double
+    description: Voltage
+  - name: Camera_aspiclTemp0
+    "@id": "#RebExposureEFD.Camera_aspiclTemp0"
+    datatype: double
+    description: Temperature in Celsius
+  - name: Camera_aspicuTemp0
+    "@id": "#RebExposureEFD.Camera_aspicuTemp0"
+    datatype: double
+    description: Temperature in Celsius
+  - name: Camera_clkHI
+    "@id": "#RebExposureEFD.Camera_clkHI"
+    datatype: double
+    description: Current in mA
+  - name: Camera_clkHV
+    "@id": "#RebExposureEFD.Camera_clkHV"
+    datatype: double
+    description: Voltage
+  - name: Camera_clkLV
+    "@id": "#RebExposureEFD.Camera_clkLV"
+    datatype: double
+    description: Voltage
+  - name: Camera_digI
+    "@id": "#RebExposureEFD.Camera_digI"
+    datatype: double
+    description: Current in mA
+  - name: Camera_digV
+    "@id": "#RebExposureEFD.Camera_digV"
+    datatype: double
+    description: Voltage
+- name: Exposure_flexdata
+  "@id": "#Exposure_flexdata"
+  primaryKey:
+  - "#Exposure_flexdata.obs_id"
+  - "#Exposure_flexdata.key"
+  constraints:
+  - name: fk_obs_id
+    "@id": "#Exposure_flexdata.fk_obs_id"
+    "@type": ForeignKey
+    description: Flex data obs_id must be an Exposure exposure_id
+    columns: ["#Exposure_flexdata.obs_id"]
+    referencedColumns: ["#Exposure.exposure_id"]
+  - name: fk_key
+    "@id": "#Exposure_flxedata.fk_key"
+    "@type": ForeignKey
+    description: Flex data key must be listed in the schema
+    columns: ["#Exposure_flexdata.key"]
+    referencedColumns: ["#Exposure_flexdata_schema.key"]
+  columns:
+  - name: obs_id
+    "@id": "#Exposure_flexdata.obs_id"
+    datatype: long
+    description: Unique identifier.
+  - name: key
+    "@id": "#Exposure_flexdata.key"
+    datatype: string
+    description: Name of key.
+  - name: value
+    "@id": "#Exposure_flexdata.value"
+    datatype: string
+    description: Content of value as a string.
+- name: Exposure_flexdata_schema
+  "@id": "#Exposure_flexdata_schema"
+  primaryKey:
+  - "#Exposure_flexdata_schema.key"
+  columns:
+  - name: key
+    "@id": "#Exposure_flexdata_schema.key"
+    datatype: string
+    description: Name of key.
+  - name: dtype
+    "@id": "#Exposure_flexdata_schema.dtype"
+    datatype: string
+    description: Name of the data type of the value.
+  - name: doc
+    "@id": "#Exposure_flexdata_schema.doc"
+    datatype: string
+    description: Documentation string.  Can include units and IVOA UCD.
```

Comment:
Units and UCDs should be broken out as separate columns.  See the TAP_SCHEMA.columns schema for guidance.
https://ivoa.net/documents/TAP/20190927/REC-TAP-1.1.html#tth_sEc4.3

Reply:
Since this schema is dynamic and intended for internal engineering use only, I don't think it will be translated to TAP Schema.

Reply:
After discussion, conceded that Gregory is correct, as usual.